MAUI/DataGrid/data-binding.md

```diff
@@ -57,6 +57,10 @@ this.sfDataGrid1.ItemsSource = table;
 {% endhighlight %}
 {% endtabs %}
 
+### CanUseFilterView

```

Comment:
Remove this topic. This is not required.

Reply:
Removed